### ISSUE TITLE
Paging

### ISSUE THREAD

**Issue by cilerler** [tom-englert/DataGridExtensions#5]
Would you please add an example of how to combine filtering and paging?

**Comment by tom-englert** [comment]
If you need paging, you have to do server side filtering.

I think you will have to implement you own [`ICollectionView`](https://msdn.microsoft.com/en-us/library/system.componentmodel.icollectionview.aspx) and pass the filters to the server.

**Comment by cilerler** [comment]
I have a _oData_ service that I'm trying to consume via datagrid.  So if I didn't miss understood you, you are saying that I have to implement middleware in between.  Please confirm.

**Comment by tom-englert** [comment]
Yes, however I'm not sure if that will work. E.g. `ICollectionView.Filter` is a simple `Predicate<object>`, not an expression, so I have no idea how that could be passed to a server. Maybe the DGX would need to be aware of this scenario, so it can pass an expression instead of a complied function...

**Comment by cilerler** [comment]
Thanks

**Comment by bytefish** [comment]
I am late to the game, but I leave it here in case someone comes across this issue. I have added an example for working with an OData Service at: 

* https://github.com/bytefish/WideWorldImporters

@tom-englert Thanks for this great library!
